src/virtual-source-buffer.js

```diff
@@ -348,6 +352,10 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
       return segmentObj;
     }, sortedSegments);
 
+    if (sortedSegments.audio.info) {
+      this.mediaSource_.changeInfo_('audio', sortedSegments.audio.info);
+    }
```

Comment:
Aren't we still going to try the append even though it will likely fail with a decode error?